### ISSUE TITLE
API Use Cases - guidance on the use of the "display" property

### ISSUE THREAD

**Issue by briandiggle** [nhsconnect/gpconnect#136]
Some guidance is required on the population of the "display" property.

NHSD CUI (common user interface) guidance (https://digital.nhs.uk/common-user-interface) provides guidance patient, practition, organisation – display should follow that guidance. 

What about the resources where there is no CUI guidance? We need to come up with guidance on what should be present in these fields.

**Comment by michaelmeasures** [comment]
@briandiggle yes we / data standards need to come-up with what is the minimum subset of information what is needed in the display fields. Note, it is important that populating this is made mandatory and that it's dynamic/calculated in nature (so it's kept up to date when the underlying resource changes). Also, we need to ensure that the DMS gets updated in this area.

**Comment by briandiggle** [comment]
@james-answer is going to elaborate where additional guidance is required

**Comment by james-answer** [comment]
The display elements of all references within the constrained fhir resources have a cardinality of 0..1 so is optional but the specification on the following page does state that the display element within the references is mandatory:

https://nhsconnect.github.io/gpconnect/development_fhir_api_guidance.html#resource-references

The page references the CUI guidence and contains a small table with basic guidence on the display format for Patient, Practitioner and Organization references.

The display field within the CareConnect profiles still have a cardinality of 0..1 so I would be inclined to leave the profiles so that the cardinality is 0..1 but update the specification to include additional guidence for references to other types of resources and make the expected format more specific rather than an example.

Missing resource reference information in table:
- Location Reference
- Schedule Reference
- Slot Reference
- Device

**Comment by briandiggle** [comment]
The current table at https://nhsconnect.github.io/gpconnect/development_fhir_api_guidance.html#resource-references includes incorrect values for "Display Format" - this needs sorting as well. E.g. CUI guidance for patient name is along the lines of "SURNAME, givenname". I would just refer to the CUI guidance (which is comprehensive) for those areas that are already covered.

**Comment by jackiebarnes** [comment]
Recommended that this table and associated guidance is removed from FHIR API Guidance page, given its inaccuracy, lack of significance and fact that no Providers have implemented it
